blog/2023-11-27-magazine-006.md

```diff
@@ -0,0 +1,117 @@
+
+Check this out:
+![mutarray_1](./assets/2023-11-27-magazine-006/singleton.png)
+
+
+❤️ [Elad Ben-Israel](https://github.com/eladb) ❤️
+
+
+### Remove `$` from `String` Interpolations
+
+#### This one is a life changer :)
+Before this change, we used the `$` for String interpolations, but not anymore! 
+The motivation behind this one is *one less char to type*, more ergonomic and the `$` char had no practical use.  
+In our context, we're using an expression inside the `string` interpolation so we need a start and end token (the curly braces) but anything else seems redundant.
+
+This is a heck big of a deal! and it probably breaks a lot of Wing apps... but we believe its really worth it!
+
+![remove_dollar_from_string_interolations](./assets/2023-11-27-magazine-006)
+
+❤️ [Yoav Steinberg](https://github.com/yoav-steinberg) ❤️
+
+
+
+### Expose AWS Resources
+
+Winglang now supports a way to cast a Wing `cloud.Bucket` or any Wing resource into its AWS counterpart. This means that you can actualy grant access to the main AWS resources within Wing Resources.
+
+![expose aws resource](./assets/2023-11-27-magazine-006/expose_aws_resource.png)
+
+❤️ [Marcio Cruz](https://github.com/marciocadev) ❤️
+
+
+### Sourcemaps and Stacktraces
+
+This change adds sourcemap generation to the compiler. These sourcemaps are used for stacktrace rewriting in both preflight and inflight. 
+Before this change, when testing code using the `expect` keyword, for instance the error message originated from JavaScript, as Wing compiles to JavaScript, and was not very clear and informative:
+![sourcemaps_and_stacktraces_before](./assets/2023-11-27-magazine-006/sourcemaps_and_stacktraces_before.png)
+
+With this change, you can now get a more informative and clear error message:
+![sourcemaps_and_stacktraces_after](./assets/2023-11-27-magazine-006/sourcemaps_and_stacktraces_after.png)
+
+❤️ [Mark McCulloh](https://github.com/MarkMcCulloh) ❤️
+
+#### And there's even more excitement waiting for you! Feel free to explore the [The 23rd Wingly Update](https://www.twitch.tv/winglangio) for an extra dose of information. Join in and enjoy!
+
+### Upcoming Events
+
+**🔥 Live Workshop - January 3rd @4:30 - 6pm IST**
+Get hands-on with [Eyal Keren](https://github.com/ekeren) as he walks us through the process of constructing a full-fledged serverless workflow that ingests PDF files into one of Bedrock's supported foundation models, allowing you to query an AI agent and ask questions about the content of the file.
+For more information and to register, [click here](https://www.eventbrite.com/e/amazon-bedrock-winglang-tickets-769562721817?aff=oddtdtcreator).
+
+### The Wingly Update
+
+**[The Wingly Update](https://www.twitch.tv/winglangio)** is our biweekly stream where we share the latest developments of the project, chat with folks from the cloud industry, geek out and celebrate the beauty of the cloud.
```

Comment:
Can we add a link to a calendar invite here so they can put it in their schedule?